### ISSUE TITLE
Missing rear packages in the obs repository

### ISSUE THREAD

**Issue by didacog** [rear/rear#2923]
There are several rear packages missing from the OBS repository / rear download page (https://relax-and-recover.org/download/).

I understand that for old distro versions there will not be any new builds, but I see no reason to remove the rear versions that were working on those systems. 

Is possible to reestablish these repos that are now missing?

debian 6, debian 7, debian 8, debian 9, ubuntu 12.x, ubuntu 14.x, ubuntu 16.x, ubuntu 17.x, ubuntu 19.x & CentOS 5 are some of the packages missing.

I know that these are old and unsupported OS versions, but for DR purposes some people may need to maintain an old OS version and may want rear package available for the version that OS was supported.

Thanks,
Didac

**Comment by jsmeix** [comment]
For those older Linux distribution versions
their build repositories settings were removed
in the openSUSE Build Service (OBS)
from the ReaR package building project
via regular requests that all maintainers
of OBS package build projects get from time to time
to free computing resources in OBS for sufficiently
building packages for current distribution versions
by no longer building for no longer maintained versions.

I don't know about OBS internals but I think
because OBS is meant (only) as a build service
(and not as a long term package download storage)
it is an automatism that packages become no longer avaiable
to be downloaded for Linux distribution versions when
they are no longer built for those distribution versions.
I think it is not possible to keep old built packages
only for download without also keeping them building.
I guess a reason is that keeping old built packages
would need too much storage space in OBS over time.

Regarding ReaR for people who have old OS vesions:

I would assume that people who have an old OS
(that is no longer maintained by those who make the OS)
and who need disaster recovery for such an old OS
already have set up their disaster recovery procedure
so they do not need to upgrade ReaR to a newer version
and normally they should not upgrade ReaR, cf. the section
"Version upgrades with Relax-and-Recover" in
https://en.opensuse.org/SDB:Disaster_Recovery

Also everybody can use our current ReaR upstream
GitHub master code via "git clone" and from there
also get any needed state before via "git checkout", cf.
"Testing current ReaR upstream GitHub master code" in
https://en.opensuse.org/SDB:Disaster_Recovery

Furthermore if someone really must have ReaR as package
I assume (but I did not test it right now) that it
usually works to install a ReaR package that is built for
a newer version on an older version of the same distribution
because ReaR is only bash scripts and documentation
so there should be always the same actual package content.
Perhaps package dependencies may not match an older OS
but then it should be possible to ignore them.

Finally if someone really must have ReaR for an old OS
built as package for his specific old OS version,
then he can build it as he needs it on his own on OBS
or he lets someone else do it for him and pays for it ;-)
